### DISCUSSION TITLE
Vanilla OS2 constantly tries to update gnome-control-center-data but fails

### DISCUSSION THREAD

**Discussion by drtreehuggster** [Vanilla-OS/live-iso#386] [Q&A] [answered]
Dear all,

VanillaOS repeatedly tried to update gnome-control-center-data but always fails. Hence it offers to update at shutdown, but seems to fail to update and does not boot on the other partition. How can I repair this behavior, or can I simply ignore it. I am a first time immutable distro user, so I am a bit lost here. 

Thanks for any advice!

Cheers,
Daniel

![Bildschirmfoto vom 2024-11-19 09-27-57](https://github.com/user-attachments/assets/2ef17a68-5329-4b12-a354-4c196d132f27)

**Comment by taukakao** [comment, marked as answer]
It's a visual bug that affects all systems. You can safely ignore it.
It will be fixed in the next couple of days.

**Reply by drtreehuggster** [reply]
Thanks for this very fast reply! It was less than 5 minutes! Cheers!

**Reply by Bohne13** [reply]
@drtreehuggster thanks for asking this question. I had the same question in mind!
@taukakao thanks for answering and looking into the "bugs", as always, damn good work! ! !
